gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/cloud/storage/contrib/nio/it/ITGcsNio.java

```diff
@@ -96,6 +96,11 @@
 
   @BeforeClass
   public static void beforeClass() throws IOException {
+    if (!System.getenv().containsKey("GOOGLE_APPLICATION_CREDENTIALS")) {
```

Comment:
I prefer to remove this. In the near future we might be able to run `ITGcsNio` with other credentials (I am thinking about Compute Engine's default credentials).